src/test/java/org/apache/maven/buildcache/LifecyclePhasesHelperTest.java

```diff
@@ -19,9 +19,9 @@
 package org.apache.maven.buildcache;
 
 import java.util.ArrayList;
```

Comment:
I notice you left the `ArrayList` import here (as opposed to in the other files), is that still needed?

Reply:
Yes, it's used in a couple of unchanged lines in this class